lib/lfrfid/protocols/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Looks like this file shouldn't be a part of PR

Reply:
My bad. Deleted.